client/src/hooks/useGameDetails.tsx

```diff
@@ -2,10 +2,11 @@ import {useNavigate} from 'react-router-dom';
 
 export const useGameDetails = () => {
     const navigate = useNavigate();
-    const createGame = (roomId: string, user: string, roomLimit: number, gameDuration: number) => navigate(`/game/${roomId}/${user}/${roomLimit}/${gameDuration}`);
+    const createGame = (roomId: string, user: string, roomLimit: number, gameDuration: number, game?: string) =>
+        navigate(`/${game ?? 'clicker'}/${roomId}/${user}/${roomLimit}/${gameDuration}`);
```

Comment:
Because the `game` prop can be string or undefined. And I think in this case this is the best way to cut out undefined values. However `||` is also could be used.